### DISCUSSION TITLE
Can I build the docs locally without evaluating all the notebooks?

### DISCUSSION THREAD

**Discussion by jordandsullivan** [unitaryfund/mitiq#2335] [Q&A]
I'm sure there is a way to do this, but I didn't see it in the [documentation](https://mitiq.readthedocs.io/en/stable/contributing_docs.html#build-the-documentation-locally). If a contributor is working on the documentation and wants to check that a fresh build is successful without rebuilding and testing all notebooks, this would be a useful option.

**Comment by jordandsullivan** [comment]
Ha! Great minds think alike. Closing as this is addressed in https://github.com/unitaryfund/mitiq/issues/2334

**Comment by purva-thakre** [comment]
![image](https://github.com/unitaryfund/mitiq/assets/66048318/24f51618-c965-49b0-aa5e-1c43bedab5ee)
